src/software/firmware/srcs/pression.cpp

```diff
@@ -7,15 +7,16 @@
  * @brief Pressure sensor related functions
  *****************************************************************************/
 
-#pragma once
 
 // INCLUDES ==================================================================
 
 // Associated header
 #include "../includes/pression.h"
 
+#ifndef UNIT_TEST
```

Comment:
Ok to keep this like that for now, but a small refactoring would probably help isolate testable code outside arduino framework. Or shall we build a mocked arduino api?

Reply:
This might be cool to have a mocked arduino API but let's start small first and put this into master ^^ Also specs have been moving really fast these past days so it is OK to take our time and refactor/test stable features.